source/Cosmos.Core/CPU.cs

```diff
@@ -62,5 +63,12 @@ public static bool DisableInterrupts()
             mInterruptsEnabled = false;
             return xResult;
         }
+
+        public void Shutdown()

```

Comment:
I think that CPU shouldn't have the Shutdown method, it's not related to the CPU, but to ACPI.